sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -240,6 +252,15 @@ export class EventHubReceiver extends LinkEntity {
       }
     };
 
+    this._onAbort = async () => {
+      const desc: string =
+        `[${this._context.connectionId}] The receive operation on the Receiver "${this.name}" with ` +
+        `address "${this.address}" has been cancelled by the user.`;
+      log.error(desc);
+     await this.close();
+     throw new Error(desc);
+    };
```

Comment:
This is for the streaming case, right? 
All errors for the streaming case are suppose to call the user's error handler. That's where the user would expect to see the error. Where does this error surface up exactly?

Reply:
Updated the code to call user's error handler.